src/items/equipment/spell_amulet_1st.json

```diff
@@ -0,0 +1,171 @@
+{
+  "_id": "vdRJy2t9VydjsxHM",
+  "name": "Spell amulet 1st",
```

Comment:
The naming scheme should match Spell Gems, which have a comma before the level iirc (e.g "Spell Amulet, 1st")